packages/mwp-tracking-plugin/src/_activityTrackers.js

```diff
@@ -13,21 +13,22 @@ type ActivityFields = {
 	referrer?: string,
 	viewName?: string,
 	subViewName?: string,
+	standardized_url?: string,
+	standardized_referer?: string,
 };
 
 export const getTrackApiResponses: TrackGetter = trackOpts => request => (
 	fields: ActivityFields
 ) => {
-	const { url = '', referrer = '', viewName, subViewName } = fields;
+	const { url = '', referrer = '', ...fieldLiterals } = fields;
 	return trackOpts.log(request, {
 		description: 'nav',
 		memberId: parseIdCookie(request.state[trackOpts.memberCookieName], true), // read memberId
 		browserId: updateId(trackOpts.browserIdCookieName)(request), // read/add browserId
 		trackId: updateId(trackOpts.trackIdCookieName)(request), // read/add trackId
 		referer: referrer,
 		url,
-		viewName,
-		subViewName,
```

Comment:
Just checking, does it matter that these two values are no longer guaranteed to be here (if they aren't in the fields in the first place)

Reply:
good check, but yes it's fine if they are not returned